examples/megacli.cpp

```diff
@@ -6724,7 +6661,7 @@ void DemoApp::request_error(error e)
     if ((e == API_ESID) || (e == API_ENOENT))   // Invalid session or Invalid folder handle
     {
         cout << "Invalid or expired session, logging out..." << endl;
-        client->locallogout(false);
+        client->locallogout(false, false);
```

Comment:
The SDK behaves differently in this case, keeping sync configs by default --> megacli should behave the same.

Reply:
Actually, we should also remove caches too for this case.
Fixed in bca6388 and aaa0ff8